src/utils/format.js

```diff
@@ -1,12 +1,12 @@
+import { crcToTc } from '@circles/timecircles';
+
 import web3 from '~/services/web3';
 
-export function formatCirclesValue(value, decimals = 2) {
+export function formatCirclesValue(value, date = Date.now(), decimals = 2) {
```

Comment:
Careful with the order of the parameters. Now that you added `date` as second parameter, anytime you call `formatCirclesValue()` with two params, the second one will be interpreted as `date` and not `decimals`. We must correct all the cases like this: https://github.com/CirclesUBI/circles-myxogastria/blob/time-circles/src/components/UBI.js#L51